src/schema/__tests__/fair.test.js

```diff
@@ -228,6 +231,23 @@ describe("Fair", () => {
     })
   })
 
+  it("includes a formatted exhibition period", async () => {
+    const query = gql`
+      {
+        fair(id: "aqua-art-miami-2018") {
+          exhibition_period
+        }
+      }
+    `
+
+    const data = await runQuery(query, rootValue)
```

Comment:
`rootValue` no longer exists in `master`, which was breaking the tests. Will fix that in my PR.

Reply:
Ah just seeing this! I already opened a PR to fix it because it was blocking one of my PRs. Should I go ahead and close it?

Reply:
Nope, that's fine! 👍